packages/odyssey-react-mui/src/MenuButton.tsx

```diff
@@ -115,6 +120,10 @@ const MenuButton = ({
   }, []);
 
   const openMenu = useCallback<MenuContextType["openMenu"]>((event) => {
+    if (onClickMenuButton) {
+      onClickMenuButton(event as React.MouseEvent<HTMLButtonElement>);
+      return;
+    }
     setAnchorEl(event.currentTarget);
   }, []);
```

Comment:
I don't know what `MenuContextType["openMenu"]` means, but we shouldn't have to ever use `as` on an `event`. The type can be set in the `useCallback` generic: `useCallback<MouseEventHandlerK<HTMLButtonElement>>`.